product_docs/docs/pgd/5.6/reference/commit-scopes.mdx

```diff
@@ -41,7 +45,24 @@ commit_scope_kind:
   | SYNCHRONOUS_COMMIT }
 ```
 
-Where `node_group` is the name of a PGD data node group.
+Where `node_group` is the name of a PGD data node group. 
+ 
+## Commit scope targets
+
+### ORIGIN_GROUP
+
+Instead of targeting a specific group, you can also use `ORIGIN_GROUP` syntax sugar, which dynamically refers to the bottommost group from which a transaction originates. Therefore, if you have a top level group, `top_group`, and two subgroups as children, `left_dc` and `right_dc`, then adding a commit scope like:
```

Comment:
Again, remove the syntax sugar reference.

Reply:
done.